src/phylum/ci/ci_gitlab.py

```diff
@@ -22,13 +23,32 @@
 SHA1_ALL_ZEROES = "0000000000000000000000000000000000000000"
 
 
+@lru_cache(maxsize=1)
```

Comment:
This is definitely a premature optimization.... But I'm not Donald Knuth, so I can overlook that.

Reply:
LOL...yeah...it also helps to ensure the value gets treated as immutable.